doc/classes/Basis.xml

```diff
@@ -390,12 +396,12 @@
 	</members>
 	<constants>
 		<constant name="IDENTITY" value="Basis(1, 0, 0, 0, 1, 0, 0, 0, 1)">
-			The identity basis. This is a basis with no rotation, no shear, and its scale being [code]1[/code]. This means that:
+			The identity [Basis]. This is an orthonormal basis with no rotation, no shear, and not scaled. This also means that:
```

Comment:
```suggestion
			The identity [Basis]. This is an orthonormal basis with no rotation, no shear, and no scale. This also means that:
```
This enumeration mixing types of descriptions still feels off, might be a better way to rewrite it

Reply:
```suggestion
			The identity [Basis]. This is an orthonormal basis with no rotation, no shear, and unit scale. This also means that:
```
- "no scale" could be misinterpreted as "zero scale",
- "not scaled" I consider better as I'd rather interpret it as "no additional scale" (on top of the default unit scale),
- "unit scale" is clear and I don't see how it could be misinterpreted.

(whatever the final wording, the same for `Transform{2D|3D}.IDENTITY`)

Reply:
Could you elaborate?

Reply:
> Could you elaborate what you mean?

I mean "unit scale" is clearer than "not scaled" which is clearer than "no scale". Not sure what do you want me to elaborate further. :upside_down_face:

Reply:
I was talking to @AThousandShips 😓

Reply:
I think "unit scale" is clearer than "no additional scale" (additional to *what*?). But I'm already pretty familiar with the term "unit length" from vectors.

Reply:
I can change it, begrudgingly. I'd like @kleonc 's opinion on it. It's bikeshedding, but if we're changing it it's gotta be for the better.

Reply:
```
The identity [Basis]. This is an orthonormal basis with no rotation, no shear, and a scale of Vector3.ONE . This also means that:
```
Something like this also seems reasonable. (I can't make actual suggestions anymore because the conversation is outdated.) 

To be clear, I don't consider "no additional scale" to be an improvement over the existing docs. While "its scale being 1" is technically incorrect, it does at least mention that the number is 1 and not 0.

IMO: "No scale" ~= "Not scaled" < "No additional scale" < "its scale being 1" < "unit scale" ~= "scale of Vector3.ONE"

Reply:
At this point I may as well mention the constant, yeah...

Reply:
I do dislike the current "no additional scale". It feels off as on its own it's lacking the "additional to what" info, as mentioned.

---

After rethinking I kinda dislike my proposed "unit scale". Firstly I want to point out that anyone understanding basics of matrices already knows what identity is, hence this description's main audience is probably "non-math" people. Thus I don't think using term "unit" is a good idea here. Also, as @tetrapod00 mentioned:

> I think "unit scale" is clearer than "no additional scale" (additional to _what_?). But I'm already pretty familiar with the term "unit length" from vectors.

AFAICT in the docs "unit" is indeed used mostly for unit-length vectors, so the "unit scale" could in fact introduce additional confusion as `(1, 1, 1)` is not a unit vector. So if anything it should rather be "unit scale per axis". Which still isn't something I'd consider easy to comprehend.

Mentioning the one(s) explicitly I think will be the easiest to undersand. So I like this suggestion:

> ```
> The identity [Basis]. This is an orthonormal basis with no rotation, no shear, and a scale of Vector3.ONE . This also means that:
> ```

Reply:
Alright then! Bikeshedding complete